pytest_doctestplus/plugin.py

```diff
@@ -323,6 +325,24 @@ def parse(self, s, name=None):
                             skip_next = True
                             continue
 
+                    if config.getoption('remote_data', 'none') != 'any':
+                        matches = [re.match(
+                            r'{}\s+doctest-remote-data\s*::(\s+.*)?'.format(comment_char),
+                            last_line) for last_line in last_lines]
+
+                        if len(matches) > 1:
+                            match = matches[0] or matches[1]
+                        else:
+                            match = matches[0]
+
+                        if match:
+                            marker = match.group(1)
+                            if (marker is None or
+                                    (marker.strip() == 'win32' and
+                                    sys.platform == 'win32')):
+                                skip_next = True
+                                continue
```

Comment:
You don't need the extra match that you copy-pasted from `doctest-requires`, as you don't need to capture the upstream dependency here.
```suggestion
                    if config.getoption('remote_data', 'none') != 'any':
                        matches = [re.match(
                            r'{}\s+doctest-remote-data\s*::'.format(comment_char),
                            last_line) for last_line in last_lines]

                        if len(matches) > 1:
                            match = matches[0] or matches[1]
                        else:
                            match = matches[0]

                        if match:
                            skip_next = True
                            continue
```

Reply:
oh I see